beets/util/__init__.py

```diff
@@ -833,7 +838,7 @@ def plurality(objs: Sequence[T]) -> T:
     return c.most_common(1)[0]
 
 
-def convert_command_args(args: List[bytes]) -> List[str]:
+def convert_command_args(args: Sequence[Bytes_or_String]) -> List[str]:
```

Comment:
The type fixes in this module are unfortunately out of scope of the `namedtuple` -> `NamedTuple` replacement. Please undo them and submit a separate PR as they need to be reviewed carefully.